components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/store-api.yaml

```diff
@@ -3780,14 +3780,19 @@ definitions:
         type: string
         description: Consumer secret of the application
       validityPeriod:
-        type: integer
+        type: string
```

Comment:
Should we make this an String?
If the requirement is to have a Long integer, we can use 
```
type: integer
format: int64
```

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/6704